Content.Server/Alert/Click/RemoveEnsnare.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Content.Server.Ensnaring;
+using Content.Server.Ensnaring.Components;
+using Content.Shared.Alert;
+using JetBrains.Annotations;
+
+namespace Content.Server.Alert.Click;
+[UsedImplicitly]
+[DataDefinition]
+public sealed class RemoveEnsnare : IAlertClick
+{
+    public void AlertClicked(EntityUid player)
+    {
+        if (IoCManager.Resolve<IEntityManager>().TryGetComponent(player, out EnsnareableComponent? ensnareableComponent))
```

Comment:
Can you not have dependencies on this?

Reply:
No it errors out if you do.

 unhandled: System.NullReferenceException: Object reference not set to an instance of an object.
   at Content.Server.Alert.Click.RemoveEnsnare.AlertClicked(EntityUid player) in J:\Program Files\SS14CloneNew\space-station-14\Content.Server\Alert\Click\RemoveEnsnare.cs:line 21